updating/updating-disconnected-cluster.adoc

```diff
@@ -1,23 +1,131 @@
 [id="updating-disconnected-cluster"]
-= Updating a disconnected cluster to a minor version
+= Updating a disconnected cluster
 include::modules/common-attributes.adoc[]
 :context: updating-disconnected-cluster
 
 toc::[]
 
-You can update, or upgrade, an {product-title} cluster that you installed in a
-restricted network to a minor version by using the web console.
+You can update, or upgrade a restricted network {product-title} cluster using the `oc` command-line interface (CLI).
 
 .Prerequisites
 
+* Have access to a container registry in the restricted-network environment to push and pull images. The container registry should be compatible with Docker registry API v2.
+* You must have the `oc` command-line interface (CLI) tool installed.
 * Have access to the cluster as a user with `admin` privileges.
 See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permissions].
 * Have a recent xref:../backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your upgrade fails and you must xref:../backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore your cluster to a previous state].
 
-//mirror module
+[IMPORTANT]
+====
+If you are upgrading from earlier release than 4.5.0, you must use the manual method of creating configmap.
 
-include::modules/update-service-overview.adoc[leveloffset=+1]
+This is because `oc` command enhancement is not available in previous releases.
+====
 
-include::modules/understanding-upgrade-channels.adoc[leveloffset=+1]
+[id="update-restricted-network"]
+== Updating a restricted network {product-title} cluster
 
-include::modules/update-upgrading-web.adoc[leveloffset=+1]
+A restricted network environment is the one in which your cluster nodes cannot access the internet. For this reason you must mirror the images to a filesystem disconnected from that environment and then bring that host or removable media across that gap.
+
+If multiple clusters present within the restricted network, the recomended way is to mirror required the release images to a single container image registry and use it for updating all the clusters. 
+
+.Procedure
+
+. Findout if there is link:https://access.redhat.com/solutions/4583231[an update path] exists between your current version and the version you want the cluster to move.
+
+. Mirror the version images to the internal container registry.
+
+** Connect the removable media to a system connected to the internet.
+** Mirror the images and configuration manifests to a directory on the removable media.
++
+----
+$ oc adm release mirror "--to-dir=${REMOVABLE_MEDIA_PATH}/mirror" quay.io/openshift-release-dev/ocp-release:{product-version}.0-x86_64
+----
++
+
+** Take the media to the disconnected environment and upload the images to the local container registry
++
+----
+$ oc image mirror "--from-dir=${REMOVABLE_MEDIA_PATH}/mirror" file://openshift/release:{product-version}.0* REGISTRY/REPOSITORY
```

Comment:
1. This make me have to raise https://bugzilla.redhat.com/show_bug.cgi?id=1806779 again here. Per my testing, `oc image mirror` does not print out `ImageContentSourcePolicy` entry.  Maybe we need to raise the bug priority, so that QE can do adjustment in time for our test cases and automation scripts to align with this new change. 
2. Here we will only support user to upload files to registry (registry-mirror-to-disk-to-registry scenario), but not support mirror image from quay.io to local registry directly  (registry-mirror-to-registry scenario) any more? In present QE's disconnected cluster upgrade testing, we mainly tested registry-mirror-to-registry scenario.

Reply:
Looks like https://bugzilla.redhat.com/show_bug.cgi?id=1806779 ON QA now. Can you test it and see if it works for you now?

Reply:
I just completed my verification some hours ago. Now looks more clear.

About how to mirror release image to private registry, there are two ways, one is registry-mirror-to-disk-to-registry, another is registry-mirror-to-registry, here we only mentioned one scenario.
And for registry-mirror-to-disk-to-registry scenario, we also need tell user https://github.com/openshift/openshift-docs/pull/17896/files#diff-21ddacc727855ec1dc3074e6e76e430dR56-R57 so that get ImageContentSourcePolicy, and applied it to connected cluster. 

I know this PR focus on upgrade part, but from customer view, this is the 1st time to mention registry-mirror-to-disk-to-registry scenario, install section did not mentioned this yet. Or do we intend to publish this one and https://github.com/openshift/openshift-docs/pull/17896 together?

Reply:
@jianlinliu I have mentioned `--from-dir` and `--to-dir` workflow. In the first iteration  I have no intention of mentioning `-from=file:` workflow.  May be in a follow up PR. I have no plans around https://github.com/openshift/openshift-docs/pull/17896.